src/api-reference/authentication/sign_in_with_concur.markdown

```diff
@@ -308,9 +308,11 @@ Depending on the products the customer has enabled, integrations and features av
   - Non-Triplink Access:
     - Travel Profile
     - E-receipt
-- Expense Only Access:
+- Expense Only Access:* 
   - E-receipt
 
+* Special Note: For TripLink Suppliers that support the [Travel Receipts API](/api-reference/travel-receipts/getting-started.html), this API does not support Expense only users. It is recommended that e-receipts be posted for these or all users via the [Receipts API](/api-reference/receipts/get-started.html).
```

Comment:
This asterisk will show up as a bullet. Please escape by using a backslash. Ex. `\*`